src/AccessibilityInsights.Extensions.AzureDevOps/ConfigurationControl.xaml.cs

```diff
@@ -249,11 +245,11 @@ private async void ChangeStates(ControlState state)
                 try
                 {
                     ToggleLoading(true);
-                    Dispatcher.Invoke(projects.Clear);
+                    Dispatcher.Invoke(VMConfig.Projects.Clear);
                     var newProjectList = await UpdateTeamProjects().ConfigureAwait(true); // need to come back to original UI thread.
-                    Dispatcher.Invoke(() => newProjectList.ForEach(p => projects.Add(p)));
+                    Dispatcher.Invoke(() => newProjectList.ForEach(p => VMConfig.Projects.Add(p)));
                     ToggleLoading(false);
-                    Dispatcher.Invoke(() => serverTreeview.ItemsSource = projects);
+                    Dispatcher.Invoke(() => serverTreeview.ItemsSource = VMConfig.Projects);
```

Comment:
With the binding working correctly, we may be able to remove the code to update the ItemsSource